provdebug/pvdebug.py

```diff
@@ -79,8 +79,11 @@ def run():
 
     provdb_command = sys.argv[1]
     if provdb_command == "-r" or provdb_command == "--replay":
-        replayer = prov.Replayer(args.file)
         print("Welcome to the Multilingual Provenance Debugger in Replayer Mode, type help for more information")
+        if not args.file.endswith(".replay"):
+            print("The following file extension for the replayer is required: .replay")
+            return
```

Comment:
Should we return _before_ we have the error message? i.e. should the tool even "start" if its given a file with the incorrect format?

Reply:
No, I don't think it should start, you're right.

Reply:
Ah by "tool start" I kinda meant the intro for the replayer gets printed, since to the user, that's the _start_ of the program. Does that make more sense?

Reply:
So something like
```sh
> provdb -r not-replay.txt
The following file extension for the replayer is required: .replay
```
instead of 
```sh
> provdb -r not-replay.txt
Welcome to the Multilingual Provenance Debugger in Replay mode
The following file extension for the replayer is required: .replay
```

Reply:
Yep! I thought you mean changing the replayer start up condition to be like:
```python
if (provdb_command == "-r" or provdb_command == "--replay") and args.file.endswith(".replay"):
...
else:
   # print error message
```
But then I think the error message would end up printing if the user starts up the debugger instead. I guess we can look into later for organizing the code from that mode into its own branch block?